spec/models/salary_frequency_spec.rb

```diff
@@ -1,12 +1,33 @@
 RSpec.describe Wpcc::SalaryFrequency do
   subject do
     described_class.new(
+      locale: locale,
       params_salary_frequency: params_salary_frequency,
       session_salary_frequency: session_salary_frequency
     )
   end
 
   let(:params_salary_frequency) { nil }
+  let(:locale) { :en }
+
+  describe '#to_adj' do
+    let(:params_salary_frequency) { 'week' }
```

Comment:
@margOnline Do you reckon that we may need at least three more test cases to cover the remaining adjectives?